politeiawww/templates.go

```diff
@@ -93,6 +97,12 @@ You are receiving this email because a password reset was initiated for {{.Email
 administrators.
 `
 
+const templateUserPasswordChangedRaw = `
+You are receiving this email to notify you that your password has changed for 
+{{.Email}} on Politeia. If you did not perform this action, please contact Politeia
```

Comment:
Perhaps we should add some guidance on *how* to contact the Politeia admins?

Reply:
Hey @jholdstock, I updated the email message with instructions to contact admins through slack. Anything else that you think we should speak with the user?

Reply:
Looks good to me